src/python/Utils/TokenManager.py

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#pylint: disable=W0212
+"""
+File       : TokenManager.py
+Author     : Valentin Kuznetsov <vkuznet AT gmail dot com>
+Description: IAM token manager
+"""
+
+# system modules
+import os
+import ssl
+import time
+import logging
+import traceback
+
+# third part library
+try:
+    import jwt
+except ImportError:
+    traceback.print_exc()
+    jwt = None
+
+# prevent "SSL: CERTIFICATE_VERIFY_FAILED" error
+# this will cause pylint warning W0212, therefore we ignore it above
+ssl._create_default_https_context = ssl._create_unverified_context
+
+
+def getToken(name=None):
```

Comment:
Now that I am getting to the end of this review. I see that we have this `getToken` function but the class below also has a `getToken` method. Even though we need a `TokenManager` object (self) to call the `getToken` method, I think it would be less confusing to rename this function to actually `readToken`.
What do you think?

Reply:
done